model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/changes/PersistentSessionsWorker.java

```diff
@@ -76,7 +76,8 @@ public void run() {
 
         private void process(ArrayBlockingQueue<PersistentUpdate> queue) throws InterruptedException {
             ArrayList<PersistentUpdate> batch = new ArrayList<>();
-            PersistentUpdate polled = queue.poll(100, TimeUnit.MILLISECONDS);
+            // Timeout of 10 seconds is only a backup if interrupting the task didn't work as expected.
```

Comment:
10 seconds does not correspond to the value below. Also, what do you mean by "if interrupting the task didn't work as expected"?